tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -58,6 +59,180 @@ public function test_add_feature() {
 		$this->assertEquals( null, $re_added_feature );
 	}
 
+	public function test_add_feature__with_tags() {
```

Comment:
Try to use the `@dataProvider` approach

Example: https://github.com/elementor/elementor/blob/5eb5cf28f37fbf2ad35cc68319ff830ca9249750/tests/phpunit/elementor/app/import-export/test-module.php#L163

https://github.com/elementor/elementor/blob/5eb5cf28f37fbf2ad35cc68319ff830ca9249750/tests/phpunit/elementor/app/import-export/test-module.php#L193